src/index.ts

```diff
@@ -10,15 +11,18 @@ import {
 const GITHUB_TOKEN = getInput('GITHUB_TOKEN');
 
 const octokit = new GitHub(GITHUB_TOKEN);
+const GITHUB_LOGIN = getInput('GITHUB_LOGIN');
```

Comment:
Should we move it before the definition of `GITHUB_TOKEN `? Seems to belong there, rather than here.

Reply:
fixed with: https://github.com/ridedott/dependabot-auto-merge-action/pull/233/commits/8fa422481dda410cd7bc52b6c04ead5cbcab9c54